modules/gathering-data-specific-features.adoc

```diff
@@ -53,10 +53,21 @@ ifndef::openshift-rosa,openshift-dedicated[]
 |Data collection for Local Storage Operator.
 
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:v<installed_version_sandboxed_containers>`
-|Data collection for {sandboxed-containers-first}.
+|Data collection for {osc}.
 
-|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
-|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
+|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed_version_NHC>`
+|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check (NHC) Operator, and the Node Maintenance Operator (NMO).
+
+Use this image if your NHC Operator version is *less than 0.9.0*. 
```

Comment:
[Per ISG use (earlier) to refer to a previous software version or fix level](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#e).

Reply:
Thanks @mburke5678 for your feedback. I have applied your suggestion.